### PR TITLE
Add edit membership

### PR DESCRIPTION
**Pull request by ashleytqy**
Position is displayed as **words** not numbers, Lead displayed first
Start and End date text box is read-only

### CODE REVIEW

**Comment by ethanresnick** [comment]
I <3 this! Thank you @ashleytqy!

One thought: Is there a way to structure this so we're not duplicating the logic for constructing a position's name/string representation (i.e. `` `${teamName} ${(isLead ? '- Lead' : '')}` ``) in a bunch of different places? If we want to add something to that string later on -- say, because we change the structure of Position -- we could miss some places.

**Comment by ashleytqy** [comment]
Welcome ~ !! Would creating a new module in the services folder (or maybe add it into data-transformer?) solve this issue?

**Comment by ethanresnick** [comment]
I'm not sure... this really isn't a codebase I know well. @brentdur would have more insight

**Comment by brentdur** [comment]
@ashleytqy could also possibly try a filter?

**Comment by brentdur** [comment]
@ashleytqy

File: app/partials/actions/membership/membership-edit.html [hunk 1]
```diff
@@ -1,16 +1,57 @@
...
 		Edit {{rdesc.attributes.name.singular}}
 	</h3>
 	<h5>{{rdesc.attributes.description}}</h5>
 
-	<form name="form" novalidate>
-		<div class="form-group"
-			ng-repeat="field in rdesc.attributes.fields"
-			ng-if="fep.hiddenFields.indexOf(field.name) == -1 && field.name !== 'isActive'"
-			ng-include="'/partials/fields/default-field.html'">
+<div class="form-group" ng-form>
```
Comment (brentdur) [inline app/partials/actions/membership/membership-edit.html]: nit: indent?

**Pull request merged**